tools/rcverify.sh

```diff
@@ -38,6 +38,8 @@ V=${3?"missing version e.g., '3.19.0'"}
 # the release candidate, usualy 'rc1'
 RC=${4:-rc1}
 
+REMOVE_DIR=${5:-true}
```

Comment:
```suggestion
REMOVE_DIR=${5:-cleanup}
```

Reply:
Hmm actually this sets the default to `true` if the 5th parameter is not given?

Reply:
Yes. I mentioned it in the PR description as well.
If we need to keep the previous behavior, would be better to change it to `false`.

But I suppose most people would not need to keep them?

Reply:
Updated according to your suggestions.
But I keep it enabled by default.
Should I change it to be disabled by default?

Reply:
My reason for keeping the directory around was to manually inspect the release directory. 
I don't object to the change, I suppose it's more convenient for most.